MM2RandoLib/Extensions/StringExtensions.cs

```diff
@@ -10,6 +15,18 @@ public static class StringExtensions
         //
         // Public Methods
         //
+
```

Comment:
I disagree. Hopefully I make some good arguments:

1. The seed contract is an interface and not a class (or abstract class).
2. The seed interface does not define/prescribe anything about the seed string or the identifier, other than they are strings (i.e. they do not necessitate any type of integer or string conversions).
3. Even if the seed class was an abstract class, it would not be inherently using those conversions methods.  I could see them being part of the derived classes, but since there could be many that use any number of conversion methods, I believe the best place for them is an external class, like a conversion class.
4. The conversion methods are independent of seed operations (i.e. they are strictly conversions between numeric types to string types and vice versa), thus, their best place is as extensions to those types.